### ISSUE TITLE
Confirm password Field?

### ISSUE THREAD

**Issue by ropoko** [arm-software-inc/tuneradio.web#1]
do we need a confirm password field?

what's the pros and cons of having one?

**Comment by abjesus** [comment]
i don't think we need a confirmation field. It's not important on the backend and can be easily validated on the frontend.
